src/app.js

```diff
@@ -208,6 +220,88 @@ gpii.app.exit = function (that) {
 };
 
 
+fluid.registerNamespace("gpii.app.pcp");
+
+/**
+ * Sends a message to the given window
+ *
+ * @param pcpWindow {Object} An Electron `BrowserWindow` object
+ * @param messageChannel {String} The channel to which the message to be sent
+ * @param message {String}
+ */
+gpii.app.pcp.notifyPCPWindow = function (pcpWindow, messageChannel, message) {
+    if (pcpWindow) {
+        pcpWindow.webContents.send(messageChannel, message);
+    }
+};
+
+/**
+ * Creates an Electron `BrowserWindow` that is to be used as the PCP window
+ *
+ * @param width {Number} The desired width of the window
+ * @param height {Number} The desired height of the window
+ * @returns {Object} The created Electron `BrowserWindow`
+ */
+gpii.app.pcp.makePCPWindow = function (width, height) {
+    var screenSize = electron.screen.getPrimaryDisplay().workAreaSize;
+    // TODO Make window size relative to the screen size
+    var pcpWindow = new BrowserWindow({
+        width: width,
+        height: height,
+        show: false,
+        frame: false,
+        fullscreenable: false,
+        resizable: false,
+        alwaysOnTop: true,
+        skipTaskbar: true,
+        x: screenSize.width - width,
+        y: screenSize.height - height,
+
+        // feel the app more native
+        backgroundColor: "#71c5ef"
```

Comment:
I think this should be pulled out into the config since it seems like something we might want to override.